src/main/java/pl/psnc/dei/service/EuropeanaRestService.java

```diff
@@ -0,0 +1,47 @@
+package pl.psnc.dei.service;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Service;
+import org.springframework.web.reactive.function.client.WebClient;
+import pl.psnc.dei.exception.DEIHttpException;
+import pl.psnc.dei.request.RestRequestExecutor;
+import reactor.core.publisher.Mono;
+
+import java.util.HashMap;
+import java.util.Map;
+
+@Service
+public class EuropeanaRestService extends RestRequestExecutor {
```

Comment:
I have doubts whether this is a proper name of the class...it suggests that it's for all communication with Europeana while in fact it's just for Record API. We have another service SearchService that in fact looks similar but is dedicated for different API that is still a Europeana API. We could make a small refactoring where we can use Europeana as prefix and change SearchService to EuropeanaSearchService and this one to EuropeanaRecordService. This would make us a bit more prepared for other aggregator where we will probably have more search and record services related to certain aggregator.

Reply:
He changed its name bc of me.
I though that it would be nice to have one class that takes care of all communication with Europeana.

Take a look at how it looks after adding transcription posting method:
https://github.com/EnrichEuropeana/dei/pull/22/files

Reply:
OK. In this case my comment is in fact not relevant. We should have this in mind while merging your branch to master.